includes/Modules/Analytics_4/Web_Tag.php

```diff
@@ -121,18 +117,15 @@ protected function render() {
 	}
 
 	/**
-	 * Enqueues gtag script.
+	 * Configures gtag script.
 	 *
 	 * @since 1.24.0
+	 * @since n.e.x.t Renamed and refactored to use new GTag infrastructure.
+	 *
+	 * @param GTag $gtag GTag instance.
 	 */
-	protected function enqueue_gtag_script() {
+	protected function setup_gtag( $gtag ) {
```

Comment:
Let's use a proper type hint for the parameter as well.
```suggestion
	protected function setup_gtag( GTag $gtag ) {
```

Reply:
Added, thank you for the tip!